stream-chat-android-client/src/main/java/io/getstream/chat/android/client/controllers/ChannelControllerExtensions.kt

```diff
@@ -8,6 +8,10 @@ import kotlin.reflect.KClass
 /***
  * Subscribes to events of type [T] in the channel.
  */
+@Deprecated(
+    message = "The ChannelController has been replaced by ChannelClient",
+    replaceWith = ReplaceWith("ChannelClient::subscribeFor")
```

Comment:
These are not valid `ReplaceWith` usages (the intention actions they generate break the code), we should remove them, as we can't provide good replacements for extensions being moved to a new package, this has to be done manually